src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  overflow: auto;
```

Comment:
The `overflow: auto;` property may not be necessary for the `.card` class. Unless you specifically need to handle overflowing content, it's better to remove it to avoid unexpected scrollbars. If you need to clear floats or contain inner elements, consider using `overflow: hidden;` or a clearfix solution.